resources/assets/components/UserIndex.js

```diff
@@ -23,12 +22,20 @@ const USER_SEARCH_QUERY = gql`
  * @param {string} term
  */
 const SearchResults = ({ term }) => {
+  const title = 'Members';
+  const subtitle = 'User profiles &amp; signups...';
+
   const { loading, error, data } = useQuery(USER_SEARCH_QUERY, {
     variables: { term },
   });
 
-  if (loading) return <div className="spinner" />;
-  if (error) return <span>Error :(</span>;
+  if (loading) {
+    return <Shell title={title} subtitle={subtitle} loading />;
```

Comment:
I don't see `Shell` imported or a component by that name in Rogue, is this something built in to React?

Reply:
Whoops, forgot to update that to be `Chrome`!

Reply:
Renamed this component to `Shell` in e44e5b53 so this now works! 😁